en/flight_modes/offboard.md

```diff
@@ -0,0 +1,42 @@
+# Offboard Flight Mode
+
+The vehicle obeys a position, velocity or attitude setpoint provided over MAVLink (often from a companion computer connected via serial cable or wifi). The setpoint can be provided by APIs like [DroneCore](http://dronecore.io/) or [MAVROS](https://github.com/mavlink/mavros).
+
+> **Note** 
+>  * This mode requires position information (GPS or optical flow).
+>  * This mode is automatic (RC control is disabled [by default](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) except to change modes).
+>  * The vehicle must be armed before this mode can be engaged.

```

Comment:
Not only that - to change to this mode, a stream of setpoint msgs should be sent prior to changing the mode.

Reply:
Can you please expand on this/why? 
Are you saying that PX4 needs to have already received setpoint messages before changing to the mode? Are those messages ignored - ie if received before the vehicle enters the mode?
I *guess* what you are saying is that offboard mode needs an active connection so you need to send messages before you switch to offboard mode or you'll immediately timeout - and the messages are ignored. Correct?

Reply:
Ping @TSC21

Reply:
> I guess what you are saying is that offboard mode needs an active connection so you need to send messages before you switch to offboard mode or you'll immediately timeout - and the messages are ignored. Correct?

That is correct. To change to this mode there needs to be an active setpoint msg stream.